sdk/metricsadvisor/ai-metrics-advisor/review/ai-metrics-advisor.api.md

```diff
@@ -0,0 +1,1152 @@
+    value: "AutoDetect" | "ChangePoint" | "NotChangePoint";
+} & MetricFeedbackCommon;
+
+// @public
+export type MetricCommentFeedback = {
+    feedbackType: "Comment";
+    startTime?: Date;
+    endTime?: Date;
+    comment: string;
+} & MetricFeedbackCommon;
+
+// @public
+export type MetricDetectionCondition = DetectionConditionsCommon;
+
+// @public
+export interface MetricEnrichedSeriesData {
+    expectedValueList?: number[];
+    isAnomalyList?: boolean[];
+    lowerBoundaryList?: number[];
+    periodList?: number[];
+    series: DimensionKey;
+    timestampList?: Date[];
+    upperBoundaryList?: number[];
+    valueList?: number[];
+}
+
+// @public
+export interface MetricFeedbackCommon {
+    readonly createdTime?: Date;
+    dimensionFilter: DimensionKey;
+    readonly id?: string;
+    metricId: string;
+    readonly userPrincipal?: string;
+}
+
+// @public
+export type MetricFeedbackUnion = MetricAnomalyFeedback | MetricChangePointFeedback | MetricCommentFeedback | MetricPeriodFeedback;
+
+// @public
+export type MetricPeriodFeedback = {
+    feedbackType: "Period";
+    periodType: "AutoDetect" | "AssignValue";
+    periodValue: number;
+} & MetricFeedbackCommon;
+
+// @public
+export class MetricsAdvisorAdministrationClient {
+    constructor(endpointUrl: string, credential: MetricsAdvisorKeyCredential, options?: MetricsAdvisorAdministrationClientOptions);
+    createAnomalyAlertConfiguration(name: string, crossMetricsOperator: AnomalyAlertingConfigurationCrossMetricsOperator, metricAlertConfigurations: MetricAlertConfiguration[], hookIds?: string[], description?: string, options?: OperationOptions): Promise<GetAnomalyAlertConfigurationResponse>;
+    createDataFeed(name: string, source: DataFeedSource, granularity: DataFeedGranularity, schema: DataFeedSchema, ingestionSettings: DataFeedIngestionSettings, options?: CreateDataFeedOptions): Promise<GetDataFeedResponse>;
+    createHook(hookInfo: EmailHook | WebhookHook, options?: OperationOptions): Promise<GetHookResponse>;
+    createMetricAnomalyDetectionConfiguration(name: string, metricId: string, wholeMetricConditions: MetricDetectionCondition, description?: string, dimensionGroupOverrideConditions?: MetricSeriesGroupDetectionCondition[], seriesOverrideConditions?: MetricSingleSeriesDetectionCondition[], options?: {}): Promise<GetAnomalyDetectionConfigurationResponse>;
+    deleteAnomalyAlertConfiguration(alertConfigurationId: string, options?: OperationOptions): Promise<RestResponse>;
+    deleteDataFeed(dataFeedId: string, options?: OperationOptions): Promise<RestResponse>;
+    deleteHook(hookId: string, options?: OperationOptions): Promise<RestResponse>;
+    deleteMetricAnomalyDetectionConfiguration(detectionConfigurationId: string, options?: OperationOptions): Promise<RestResponse>;
+    readonly endpointUrl: string;
+    getAnomalyAlertConfiguration(alertConfigurationId: string, options?: OperationOptions): Promise<GetAnomalyAlertConfigurationResponse>;
```

Comment:
I wonder for all these get/delete methods if it's worth having such a verbose parameter name. Like wouldn't `id` be enough to name the thing? It's pretty clear from the method name what id it is and there is only one.

Reply:
`id` sounds good enough for get/update/delete.  Changing